src/python/scripts/dx-fetch-bundled-depends

```diff
@@ -41,4 +41,4 @@ if 'DX_JOB_ID' in os.environ:
             if dep['id']['$dnanexus_link'].startswith('file-'):
                 print "*** Downloading bundled file", dep['name']
                 dxpy.download_dxfile(dep['id'], dep['name'])
-                subprocess.check_call(['dx-unpack', dep['name']])
+                subprocess.check_call(['dx-unpack', re.escape(dep['name'])])
```

Comment:
I tried testing it out and it throws exception if not escaped. Here is a snippet of the test:

>>> f_name="space in name.tar.gz"
>>> subprocess.check_call(['dx-unpack', f_name])
>>> Unpacking space to /home/test/tmp
 *   dx_helpers.sh, line 20: space does not exist
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['dx-unpack', 'space in name.tar.gz']' returned non-zero exit status 1